lib/build-secrets/populate.sh

```diff
@@ -1,8 +0,0 @@
-#!/bin/bash
-set -e
-
-while read -r name content; do
-	mkdir -p $(dirname "$name")
-	echo  "$content" | base64 -d > "$name";
-done <<< $(jq --raw-output '. as $top | keys[] as $k | .[$k].files | keys[] as $f | "\($top[$k].tmpDirectory + "/" + $f) \(.[$f])"' secrets.json)
-
```

Comment:
`populate.sh` and `remove.sh` were moved to `balena-io-modules/balena-multibuild-scripts`.